src/components/rows/SevenStory7s/index.js

```diff
@@ -11,22 +11,20 @@ export const SevenStory7s = ({ collection, Icon, BgPattern = "bg-diamond-dark" }
   const MaxNumberOfThumbnailStories = 6;
   return (
     <div styleName={`seven-story-7s ${BgPattern}`}>
```

Comment:
do you think we need to change the BgPattern naming ?

Reply:
bg pattern is not needed , have removed it